1.0.1 changes

cs-abhishek-shukla opened the pull request.

cs-abhinav-goel commented on README.md:
```diff
@@ -1,25 +1,25 @@
+- **Version**:  1.0.1
+- **Certified**: Yes
 - **Publisher**: Fortinet
 - **Compatible Version**: FortiSOAR v7.2.0 and above
```
FSR 7.2.1 and above

cs-abhinav-goel commented on docs/setup.md:
```diff
@@ -1,20 +1,65 @@
-| SOAR Framework     | Required for Incident Response modules and Action playbooks |
+| Solution Pack Name    | Purpose                                                                                                                                                                                           |
+|:----------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SOAR Framework v2.0.0 | <ol><li>Required for Incident Response modules and Action playbooks</li><li>Required for "Was personal data affected?" field which acts as a trigger point for GDPR Framework Playbooks</li></ol> |
```
It is coming in single line instead of coming in next line
![image](https://user-images.githubusercontent.com/24773177/193516690-1bb8fe3a-e868-45ae-a7c6-902244501327.png)

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -1,79 +1,109 @@
+    3. Similarly, to change the test email ID change value of the **testEmail** variable
+
+        ![Execution Mode Configuration Parameters](res/execution-mode-configuration-parametes.png)
+
```
@cs-abhishek-shukla  : There is also the mode can be said as "Pre-production mode", where actual emails goes to the DPO email given but in case of the DPA it fetches the actual DPA email and append "_dummy" at the end.

@cs-rohit-khune  : can you please provide the doc input for the same.

cs-abhinav-goel replied:
We also need to mention the working of the SLA PB when the Demo Mode is true and when production mode is on

cs-abhinav-goel commented on README.md:
```diff
@@ -1,25 +1,25 @@
 
-This solution pack provides a set of paybooks that enables users to gather all the necessary information and send it over to supervisory authority in time.
+This solution pack comes with a set of playbooks that help users gather all the necessary information and send it over to the supervisory authority in time.
 
```
We have not mentioned the details about the SLA PB and its configurable values
@cs-rohit-khune Please provide

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -1,79 +1,109 @@
+
+        ![Execution Mode Configuration Parameters](res/execution-mode-configuration-parametes.png)
+
+
```
@cs-abhishek-shukla We also need to add screen shot when user moves to Production mode 
![image](https://user-images.githubusercontent.com/24773177/193594901-4e4adddc-621b-4e3b-b80b-ca989e382f6b.png)

this how Manual input pop up appears while creating GDPR compliance record

cs-abhinav-goel replied:
@cs-rohit-khune : please provide input for the same

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -1,79 +1,109 @@
+- To change execution mode or test email-ID, follow the below procedure:
+    1. Goto to the *Create Data Compliance Record* playbook under the **10 - SP - GDPR Framework** collection and open the *Configuration* step
+
+        ![Execution Mode Configuration](res/execution-mode-configuration.png)
```
The image and text ix coming on the line 
![image](https://user-images.githubusercontent.com/24773177/193598876-867455e9-5fb2-4733-ad79-c24357a0e869.png)

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -1,79 +1,109 @@
+
+        ![Execution Mode Configuration](res/execution-mode-configuration.png)
+
+    2. To change of playbooks execution mode from *Test Mode* to *Production*, Modify the value of the **testMode** variable to `false`
```
Pt#2 & Pt#3 should be together, unless user does not changes both value the mode will not change

cs-abhinav-goel commented on docs/usage.md:
```diff
@@ -1,79 +1,109 @@
-  - Open the `Review and Approve Breach Report by DPO` Task and click on the 'Notify Data Protection Officer' Button to get the information reviewed and approve the report to send to Data Protection Authorities
+### Notify Data Protection Officer
+
+- Open the `Notify Data Protection Officer` task and click *Notify Data Protection Officer* button to get the information reviewed and approve the report to send to Data Protection Authorities
```
@cs-abhishek-shukla @cs-rohit-khune  We have not mentioned what details will be fill after "Notify Data Protection Officer" is filled.

cs-abhinav-goel commented on release_notes.md:
```diff
@@ -0,0 +1,15 @@
+# What's New
+
+- This solution pack is now **Certified**
+- **Summary** tab contains more details and now has the following sections available as part of this solution pack's details:
```
@cs-rohit-khune Can you please mention where I can see these details in the Incident/Data Compliance modules

cs-abhinav-goel commented on release_notes.md:
```diff
@@ -0,0 +1,15 @@
+
+- This solution pack is now **Certified**
+- **Summary** tab contains more details and now has the following sections available as part of this solution pack's details:
+    - A new category **Compliance and Reporting** &ndash; helps with filtering from the left pane when using the check-boxes under **Category**
```
@cs-rohit-khune Same for this.

cs-abhinav-goel commented on release_notes.md:
```diff
@@ -0,0 +1,15 @@
+    - Added instructions to configure **Code Snippet** connector before execution of GDPR playbooks
+- A new module **Data Compliance** has been introduced
+- A new Navigation menu was added for **Data Compliance**
+- Added new playbooks to **Check for SLA violation** and to **Send Reminder for SLA Breach**
```
PB names are "Check for SLA violation" & "Reminder for SLA Breach"

cs-abhinav-goel commented on release_notes.md:
```diff
@@ -0,0 +1,15 @@
+- Added new playbooks to **Check for SLA violation** and to **Send Reminder for SLA Breach**
+- Updated **GDPR Assessment Report** to be compatible with the newly created module i.e **Data Compliance**
+- Added a provision to execute GDPR playbooks either in **Test Mode** or in **Production Mode**
+- The task and the form associated with **Notify Affected Individuals** now also show the list of email IDs of the Affected Individuals
```
Added schedule for SLA notification

cs-abhinav-goel replied:
@cs-abhishek-shukla @cs-rohit-khune

cs-abhinav-goel replied:
Also need to mention the Usage of SLA schedule in the Usage/Configuration doc
@cs-rohit-khune @cs-abhishek-shukla

cs-abhinav-goel reviewed (changes requested):
@cs-abhishek-shukla @cs-rohit-khune  : Please do the changes suggested

cs-abhinav-goel commented on release_notes.md:
```diff
@@ -0,0 +1,15 @@
+- Added a provision to execute GDPR playbooks either in **Test Mode** or in **Production Mode**
+- The task and the form associated with **Notify Affected Individuals** now also show the list of email IDs of the Affected Individuals
+
+>**NOTE**: Users of GDPR solution pack v1.0.0 must remove the fields and picklists mentioned in [Prerequisites](docs/setup.md#prerequisites)
```
Added new picklist "GDPRReminderStatus"

cs-abhishek-shukla committed:
2.0.0 Changes
- Fixed some links
- Added dependency on Code Snippet Connector under setup.md
- Added latest doc inputs from Rohit
